src/util/WidgetBinding.ts

```diff
@@ -0,0 +1,20 @@
+const dataLayers = {}
+const configurations = {}
+
+export function updateBindedStores({ dataLayer, widgetId, store, actor, config }) {
+	if (!dataLayers.hasOwnProperty(dataLayer)) {
+		dataLayers[dataLayer] = {};
+	}
+
+	dataLayers[dataLayer][widgetId] = { store, actor };
+	configurations[widgetId] = config;
+
+	for (let [id, instance] of Object.entries(dataLayers[dataLayer])) {
+		if (id != widgetId &&
+			instance.actor &&
+			!_.isEqual(instance.store.searchResults, store.searchResults) &&
```

Comment:
Just to use this function do we really need to install lodash? I think you can create a method in helper.ts

Reply:
I prefer using this function from lodash, since custom implementation had some issues in the past. The code is practically identical to the one used in the Vue SDK...